cumulus/pallets/executive/src/lib.rs

```diff
@@ -224,131 +224,57 @@ where
 {
 	/// Execute all `OnRuntimeUpgrade` of this runtime, and return the aggregate weight.
 	pub fn execute_on_runtime_upgrade() -> frame_support::weights::Weight {
-		<(COnRuntimeUpgrade, AllPalletsWithSystem) as OnRuntimeUpgrade>::on_runtime_upgrade()
+		frame_executive::Executive::<
+			System,
+			Block,
+			Context,
+			UnsignedValidator,
+			AllPalletsWithSystem,
+			COnRuntimeUpgrade,
+		>::execute_on_runtime_upgrade()
```

Comment:
This type could have been an alias scoped to this impl block to make it a bit more readable

Reply:
Could you make an example? Not sure how to do it...

Reply:
Did some experiments, doesn't result in anything better, nevermind